### DISCUSSION TITLE
EP and Windows 11 24h2

### DISCUSSION THREAD

**Discussion by tistoubapt** [valinet/ExplorerPatcher#3650] [Q&A]
Hello

I haven't read everything, sorry
Is it planned (if any idea of when) that EP will be compatible with W11 24h2 (for the taskbar) ?

I saw this "_The reimplemented taskbar has been released for 22H2, 23H2, and 24H2._"

But it's still not done it seems "_For now, this is only available for builds 22621, 22631, and 22635. Other builds will not have the option._"

Thanks

**Comment by pyrates999** [comment]
It is compatible with the EP implementation of windows 10 taskbar.

**Reply by tistoubapt** [reply]
Ah ok, I'll test again, with the latest version, it didn't work with 24h2 (26100.xxxx)

Thanks

**Reply by pyrates999** [reply]
can you post the full build number?  You should be using the latest pre-release when testing.

**Reply by tistoubapt** [reply]
I test this one : 22621.3527.65.5
And W11 24h2 26100.xxxx

Thanks

EDIT : for takbar W10, I was talking

**Reply by exequielross** [reply]
My own experience? I was able to reimplement the taskbar on a Windows 11 IoT LTSC 2024 (24H2, 26100.1 base build), that was previously stripped down with tiny11 builder. The base ISO was the one available at massgrave's site, the one leaked from a OEM. I regreted that install because the vintage "quick launch" bar (%appdata%/Microsoft/Internet Explorer/Quick Launch path) that I always add to my installs is a little broken: when you put that bar at the left, the right pane, the one that shows the opened windows, only shows icons, not the full tile with window name. That's issued other place here in the issues, i'm not complaining, im just refering that issue, it's a known bug already (Quick Launch toolbar. #3608).
So, i went back to a win11 IoT Enterprise 23H2 (again, downloaded from massgrave's site, and again, stripped down with tiny11 builder script), that has the vanilla old taskbar windows implementation, i feel more confident using it. (additional data: I enhanced the experience using EP +7tt + OldNewExplorer in 23H2, i don't remember if i tested them on 24H2).
BUT, again, the EP own implementantion of w10 taskbar works on win11 24H2 IoT LTSC 2024 (26100.1).

**Reply by pyrates999** [reply]
Please post the full build number.  Not 26100.xxx.  You can find this by running winver.  Please include if you are running x86-64 or Arm.

**Reply by tistoubapt** [reply]
Oh ok, Windows 11 Pro x64 26100.1591

**Reply by pyrates999** [reply]
Install the latest pre-release of EP then as it has fixes for 24h2.

**Reply by tistoubapt** [reply]
It's this one 22621.3527.65.5 ?

**Reply by pyrates999** [reply]
No, check here: https://github.com/valinet/ExplorerPatcher/releases

And read the release notes on what changes you need to make to your Antivirus software.

**Reply by tistoubapt** [reply]
Ah yes, this one 22621.3880.66.5

Thanks

**Comment by tistoubapt** [comment]
Sorry for the late reply
It's ok, it works

Thanks

**Reply by pyrates999** [reply]
New release is out 66.6, please update to it.
